services/wallet/service.go

```diff
@@ -38,6 +39,7 @@ func NewService(db *sql.DB, rpcClient *rpc.Client, accountFeed *event.Feed, open
 
 // Service is a wallet service.
 type Service struct {
+	db                    *sql.DB
```

Comment:
Service accepts a `*sql.DB`, but then only uses it to generate an `accounts.Database`, perhaps `Service` should accept just the `accounts.Database`

Reply:
It doesn t only use for generating an accounts.DB but I pass specifically the accounts db being build at the layer above